armeria-backend/zio/src/main/scala/sttp/client3/armeria/zio/zio.scala

```diff
@@ -0,0 +1,62 @@
+package sttp.client3.armeria
+
+import _root_.zio._
+import sttp.capabilities.Effect
+import sttp.capabilities.zio.ZioStreams
+import sttp.client3._
+import sttp.client3.impl.zio.{ExtendEnv, SttpClientStubbingBase}
+
+package object zio {
+
+  // Forked from async-http-client-backend/zio
+  // - Removed WebSocket support
+
+  /** ZIO-environment service definition, which is an SttpBackend. */
+  type SttpClient = SttpBackend[Task, ZioStreams]
+  type SttpClientStubbing = SttpClientStubbing.Service
```

Comment:
maybe we can get rid of the `.Service` here as well?

Reply:
Maybe. This will require a deeper understand of the code base. Will see what I can find out.

Reply:
Removed .service here.